src/views/ChooseProfile.vue

```diff
@@ -1,14 +1,71 @@
 <template>
     <div>
         <authenticated-header></authenticated-header>
+        <h6 v-if="profiles.length === 0" class="title has-text-centered has-text-white">Seems, you don't have active
+            profiles</h6>
+        <h6 v-else class="title has-text-centered has-text-white">That's your profiles</h6>
+        <hr>
+
+        <p v-if="profiles.length === 0" class="subtitle has-text-centered has-text-white">Let's do one of the
+            following:</p>
         <div id="main-container">
             <div class="card">
                 <nav class="panel">
-                    <p class="panel-heading">Choose profile from the list:</p>
-                    <a class="panel-block" v-for="profile in profiles" v-on:click="chooseProfile(profile)">
-                        <span class="panel-icon"><i class="fas fa-user" aria-hidden="true"></i></span>
-                        {{profile.profileName}} (created {{profile.creationDate | formatDate}})
-                    </a>
+                    <div v-if="profiles.length === 0">
+                        <b-collapse class="card" animation="slide" aria-id="schoolCreation"
+                                    :open.sync="schoolCollapseOpen">
+                            <div
+                                    slot="trigger"
+                                    slot-scope="props"
+                                    class="card-header"
+                                    role="tab"
+                                    aria-controls="schoolCreation">
+                                <p class="card-header-title">
+                                    Create school
+                                </p>
+                                <a class="card-header-icon">
+                                    <b-icon
+                                            :icon="props.open ?  'menu-up' : 'menu-down' ">
+                                    </b-icon>
+                                </a>
+                            </div>
+                            <div class="card-content">
+                                <div class="content">
+                                    <CreateSchool/>
+                                </div>
+                            </div>
+                        </b-collapse>
+                        <b-collapse class="card" animation="slide" aria-id="schoolCreation"
```

Comment:
Can we have invitation window available regardless of amount of profiles the account has? It would be more convenient for both usage and testing purposes